### ISSUE TITLE
Is this compatible with macOS?

### ISSUE THREAD

**Issue by chrisseaton** [clupprich/ruby-build-action#6]
Thank you very much for creating this action!

I'm running to run it on macOS, and I get this error:

```
  ##[error]ENOENT: no such file or directory, open '/etc/lsb-release'
  ##[error]Node run failed with exit code 1
```

Is anyone else experiencing this?

**Comment by clupprich** [comment]
Hej @chrisseaton! You're right, macOS isn't supported right now. Let me see if I can get something running over the weekend.

**Comment by clupprich** [comment]
@chrisseaton Just released v1.3.0 which adds support for macOS.

**Comment by chrisseaton** [comment]
Thank you very much!
